pkg/webreg/webreg.go

```diff
@@ -827,6 +872,81 @@ func setWorkflowGraph(t *template.Template, chains registry.ChainByName, workflo
 		})
 }
 
+type environmentLine struct {
+	Documentation string
+	Default       *string
+	Steps         []string
+}
+
+type environmentData struct {
+	Items map[string]environmentLine
+	Type  string
+}
+
+func getEnvironmentDataItems(worklist []api.TestStep, registryRefs registry.ReferenceByName, registryChains registry.ChainByName) map[string]environmentLine {
+	var data map[string]environmentLine
```

Comment:
Unless it's important and semantically meaningful to have a `nil` here, just initialize something that can be inserted into:

```go
data := map[string]environmentLine{}
```